build.gradle

```diff
@@ -12,10 +12,16 @@ subprojects { subproject ->
     subproject.apply plugin: 'java-library'
     subproject.apply plugin: 'com.github.johnrengelman.shadow'
     subproject.apply plugin: 'maven-publish'
+    subproject.apply plugin: 'java'
 
     sourceCompatibility = '1.8'
     targetCompatibility = '1.8'
 
+    compileJava.options.encoding = 'UTF-8'
+    tasks.withType(JavaCompile) {
+        options.encoding = 'UTF-8'
+    }
+
```

Comment:
Why is this needed?  Ran tests on my machine without it and they all passed.

Reply:
Okay. Sound fine for me. I had just applied  the settings like in fromhttps://stackoverflow.com/questions/38390206/assertj-wrong-failure-when-comparing-two-unicode-strings

Reply:
Yeah, I think this will mask the issue we are resolving here.  That is - the code should work regardless of the encoding of the underlying OS.

Reply:
When i remove the config the test fails. (i work with a windows machine)

Reply:
Particular tests?  I'm on a mac.

Reply:
If it's a test - perhaps the writing of the "test" files needs to use UTF-8.

Reply:
I run agains a Windows machine on CI too.  Would be interesting to see if it also fails there.

Reply:
I was my bad. my test was wrong. i removed the config